azure-pipelines.yml

```diff
@@ -9,12 +9,6 @@ trigger:
 
 strategy:
   matrix:
-    linux_node_min_version:
-      imageName: 'ubuntu-20.04'
-      node_version: 16.18.0
-    linux_node_lts_16:
-      imageName: 'ubuntu-20.04'
-      node_version: 16.x
     linux_node_latest:
```

Comment:
Probably I'm missing something, but shouldn't we label this strategy `linux_node_min_version` and/or eventually `linux_node_lts_18` and for `linux_node_latest` to use the actual latest NodeJS `19.4.0`

Reply:
Yes, will be tackled in https://github.com/SAP/ui5-fs/pull/480